pyproject.toml

```
@@ -29,16 +29,28 @@ kaleido = "0.2.1"
 plotly-resampler = "^0.8.3.1"
 pytorch-lightning = "^1.9.4"
 tensorboard = "^2.11.2"
-torch = "^1.13.1"
+torch = "^2.0.0"
```

```suggestion
torch = "^2.0.1"
```
there is torch 2.0.1 available already